README.md

```diff
@@ -71,35 +45,3 @@ Some projects license documentation and other forms of content under
 [Creative Commons Attribution 4.0](http://creativecommons.org/licenses/by/4.0/).
 
 See specific projects to understand the license used.
-
-## Understanding the relationship between .NET Core and the .NET Framework
```

Comment:
Good q but it's all in docs in one place or another. We could link to specific docs if you found that content compelling.

It was useful. I wrote it all ;)